src/entities/config.ts

```diff
@@ -24,6 +24,7 @@ export class Config extends BaseEntity {
   /**
    * @deprecated Use `Config.getValueAs(key, 'boolean', true)` instead
    */
+  /* istanbul ignore next */
```

Comment:
Does this method still get the deprecated warning with this comment between it?

Reply:
Yep, I checked that